docs/overview/office-add-ins.md

```diff
@@ -1,152 +1,130 @@
 
-The basic components of an Office Add-in are an XML manifest file and your own web application. The manifest defines various settings, including how your add-in integrates with Office clients. Your web application needs to be hosted on a web server, or web hosting service, such as [Microsoft Azure](../publish/host-an-office-add-in-on-microsoft-azure.md).
+**Task pane**
+![Task pane](../../images/task-pane-overview.jpg)
 
+### Create new objects in Office documents 
 
-**Manifest + webpage = an Office Add-in**
-![Manifest plus webpage equals Office Add-in](../../images/DK2_AgaveOverview01.png)
+You can embed web-based objects within Excel and PowerPoint documents – called content add-ins. With content add-ins, you can integrate rich, web-based data visualizations, media (such as a YouTube video player or a picture gallery), and other external content.
 
-###Manifest
+**Content add-in**
 
+![In content add-in](../../images/DK2_AgaveOverview05.png)
 
-The manifest specifies settings and capabilities of the add-in, such as the following:
-    
-- The add-in's display name, description, ID, version, and default locale.
-    
-- How the add-in integrates with Office: 
-      - For add-ins that extend Word/Excel/PowerPoint/Outlook: The native extension points the add-in uses to expose functionality, such as buttons on the ribbon. 
-      - For add-ins that create new embeddable objects: The URL of the default page that is loaded for the object.
-       
-    
-- The permission level and data access requirements for the add-in.
-    
-For more information, see [Office Add-ins XML manifest](../../docs/overview/add-in-manifests.md).
+## Outlook add-ins that extend functionality 
 
+Outlook add-ins can extend the Office ribbon and also display contextually next to an Outlook item when you're viewing or composing it. They can work with an email message, meeting request, meeting response, meeting cancellation, or appointment when a user is viewing a received item or replying or creating a new item. 
 
-###Web app
+Outlook add-ins can access contextual information from the item, such as address or tracking ID, and then use that data to access additional information on the server and from web services to create compelling user experiences. In most cases, an Outlook add-in runs without modification on the various supporting host applications, including Outlook, Outlook for Mac, Outlook Web App, and Outlook Web App for devices, to provide a seamless experience on the desktop, web, and tablet and mobile devices. 
 
-The minimal version of a compliant web app is a static HTML webpage. The page can be hosted on any web server, or web hosting service, such as [Microsoft Azure](../publish/host-an-office-add-in-on-microsoft-azure.md). You can host your web app on the service that you choose.  
+For an overview of Outlook add-ins, see [Outlook add-ins overview](https://docs.microsoft.com/en-us/outlook/add-ins/). 
```

Comment:
I think we need this link here, because we do in fact want people who want to learn about Outlook add-ins to jump off and go read the content on the Docs site.